### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by Tozapid** [strongbox-password-safe/Strongbox#792]
**Preliminaries**
I have ensured that:

- [x] I am running the latest version of Strongbox on the App Store by searching for Strongbox and clicking into it to see the Update button (or not)
- [x] I have performed a full restart of my device no matter how annoying that is

**Versions**
 - OS: macOS Sequoia 15.0 Beta 3 (24A5289h)

**Strongbox Version**
 - Version: 1.60.8

**Describe the bug**
Since the last update, I noticed that the password generation function in the menu bar is gone. It was very convenient and I did not find any information about disabling this function in the Release Notes

**To Reproduce**
1. Click on the Strongbox icon in the Menu Bar
2. See error

**Expected behavior**
The feature was added earlier in this Feature request and the correct behavior is described there.
https://github.com/strongbox-password-safe/Strongbox/issues/442

**Screenshots & Recordings**
<img width="345" alt="image" src="https://github.com/user-attachments/assets/7e69e83e-6a35-48da-91bd-fd3e25310c4e">

**Comment by strongbox-mark** [comment]
Hi @Tozapid, probably not what you wanted to hear, but we decided to redesign the tray icon functionality and move the password generator into it's own window. We've got the new tray icon experience coming in the next release and there will be a global hotkey so that you can launch it from anywhere. It should be even more convenient to launch and grab a quick password from anywhere.

**Comment by Tozapid** [comment]
Thank you for your answer!
I'm not sure I have free space on my keyboard for another hotkey and that it would work well in fullscreen mode. Maybe I could add this interface as a separate option?
Why not make this menu customizable?

**Comment by strongbox-mark** [comment]
Yes, definitely something we could think about, let's keep open here...

**Comment by strongbox-mark** [comment]
Hi @Tozapid - You should have the option now to restore the password generator from the tray with 1.60.10. Right click on the Tray icon > Configure Click.

Let me know if you have any trouble.

**Comment by Tozapid** [comment]
Amazing! Exactly what I needed
